phpstan.neon

```diff
@@ -0,0 +1,10 @@
+parameters:
+    level: 5
+    reportUnmatchedIgnoredErrors: false
+    paths:
+        - src
+    bootstrapFiles:
+        - phpstan-bootstrap.php
+    excludePaths:
+        - src/Migrations/*
```

Comment:
```suggestion
```
@aryaantony92  I think the Migrations should also be checked. What do you think?

Reply:
Yes, we can do